sdk/core/Azure.Core/src/CompositeClassifier.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// Composes a response classifier applying customization precedence rules as follows:
+    ///   - First, try any per-invocation customizations
+    ///   - Then, try any per-client customizations
+    ///   - Finally, apply default logic for the operation using the base classifier, which
```

Comment:
I think these tweaks make it slightly clearer that we only continue to the next step if no result was found on the current step.
```suggestion
    ///   - First, try any per-invocation customizations. If any are found, return the classification result.
    ///   - Then, try any per-client customizations. If any are found, return the classification result.
    ///   - Finally, apply default logic for the operation using the base classifier, which
```